docs/internal/continuous-profiler.md

```diff
@@ -0,0 +1,255 @@
+# Continuous profiler
+
+> [!IMPORTANT]  
+> Continuous profiler is an experimental feature. It will be subject to change,
+> when <https://github.com/open-telemetry/oteps/pull/239> or <https://github.com/open-telemetry/oteps/pull/237>
+> will be ready and merged.
+
+## Thread sampling
+
+Thread sampling can be enabled by the custom plugin.
+Plugin is responsible for parsing dense, data format and exporting it in
+the appropriate format.
+
+### How does the thread sampler work?
+
+The profiler leverages [.NET profiling](https://docs.microsoft.com/en-us/dotnet/framework/unmanaged-api/profiling/)
+to perform periodic call stack sampling. For every sampling period, the runtime
+ suspended and the samples for all managed thread are saved into the buffer,
+ then the runtime resumes.
+
+The separate managed-thread is processing data from the buffer and export it
+the way defined by the plugin.
+
+To make the process more efficient, the sampler uses two independent buffers to
+store samples alternatively.
+
+### Requirements
+
+* .NET 6.0 or higher (`ICorProfilerInfo12` available in runtime). Technically
+  it could be `ICorProfiler10`, and .NET Core 3.1/.NET 5.0, but these versions
+  are not supported by OpenTelemetry .NET AutoInstrumentation.
+* .NET Framework is not supported. `ICorProfiler10` nor `ICorProfiler12`
+  are not available in .NET Fx.
```

Comment:
```suggestion
* .NET 6.0 or higher (`ICorProfilerInfo12` available in runtime).
* .NET Framework is not supported, as `ICorProfiler10` and `ICorProfiler12`
  are not available in .NET Fx.

Note that `ICorProfiler10`  and .NET Core 3.1 or .NET 5.0 are not supported
by the OpenTelemetry .NET Automatic Instrumentation.
```

Reply:
Do we expect folks to use ICorProfiler10 despite the versions it comes within being not supported?

Reply:
More like:

```
Note that `ICorProfiler10` could be supported, but .NET Core 3.1 or .NET 5.0 aren't
by the OpenTelemetry .NET Automatic Instrumentation.
```

Reply:
Suggestion updated!